packages/account/src/Containers/account.jsx

```diff
@@ -119,6 +119,10 @@ const Account = ({
             if (route.path === shared_routes.proof_of_identity || route.path === shared_routes.proof_of_address) {
                 route.is_disabled = !should_allow_authentication;
             }
+
+            if (route.path === shared_routes.proof_of_ownership) {
+                route.is_disabled = is_virtual;
```

Comment:
this is related to disabling the entry in the profile page, is it right?
in that case if client doesn't need POO dont we have to disable this too?

Reply:
Yes, but we have the page attached, that means it won't be viewed. Is that OK? Because you can only view it if you navigate to the POO menu, if I disable the menu, that means this page won't be seen. Should I disable it anyway?
<img width="1722" alt="none needed" src="https://user-images.githubusercontent.com/104334373/177705831-75b3d861-45a0-4a65-bb38-f03018bf0beb.png">

Reply:
so in case of virtual account you will not show the entry even. is it right?

Reply:
It'll be there, but disabled. Is that OK?
![Uploading virtual.png…]()

Reply:
i think its ok to place it there disable. 
in that case we have a consistent implementation, nothing changes unexpectedly and client would know where he can find the proof of ownership because he saw it already in the account page.